frontend/src/components/Analysis/AnalysisDrawer/DownloadAnalysisButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Button, ButtonProps, Tooltip } from "antd";
+import { DownloadOutlined } from "@ant-design/icons";
+
+export function DownloadAnalysisButton(props: ButtonProps) {
+  return (
+    <Tooltip
+      title={<div>Download all the charts.</div>}
```

Comment:
It's a bit weird for the button to say "Download Analysis" and then the tooltip say "Download all the charts". I see two solutions:

1. Just rename the button "Download Charts".
2. Also zip up and download other information (e.g. latex or csv tables of raw numbers) and keep the button being "Download Analysis".

I would suggest that in the short term, we take approach "1." but add an issue noting that "2." would be useful in the future.

Reply:
Thanks for the advice. I also plan to generate a report and download it together in the zip file later. In this PR, we can name it as "Download Charts" and change the name in following PRs.